src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -43,23 +50,31 @@ class NotifyService(
         "comName" to comName,
         "crn" to crn,
       )
-      sendEmail(variationForApprovalTemplateId, notifyRequest.email, values, null)
-      log.info("Notification sent to ${notifyRequest.email} VARIATION FOR APPROVAL submitted by $comName for $licenceId $firstName $lastName with crn $crn")
+      if (sendEmail(variationForApprovalTemplateId, notifyRequest.email, values)) {
```

Comment:
I've tweaked all these as they incorrectly logged that the email was sent event when it was not